pkg/dmi.go

```diff
@@ -0,0 +1,169 @@
+package pkg
+
+import (
+	"bufio"
+	"fmt"
+	"os/exec"
+	"strings"
+)
+
+type Entry = map[string][]string
+type DMIMap = map[string]Entry
+
+type DMIDecoder struct {
+	installed  bool
+	cmdOutput  string
+	decodedMap DMIMap
+}
+
+// NewCheckerService creates new instance from the parser
+func NewDMIDecoder() DMIDecoder {
+	dmidecode := exec.Command("sudo", "dmidecode")
+	res, _ := dmidecode.Output()
```

Comment:
replace `res` with `out`, typically when people read `res` the assume it's the `status code` or the `return code` which is an `int`, also don't ignore the error, if error happens it needs to be handled or propagated to the user e.g in a container and sudo isn't installed